### PR TITLE
Book store init

### PR DESCRIPTION
**Pull request by lorenzoworx**
In this pull request, I lay the foundations for our Bookstore website. I have created the user interface using React.

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @lorenzoworx,

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:

- [ ] Some linters checks are not passed. Please make sure that all checks are passing and green.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: src/App.css [hunk 1]
```diff
@@ -1,38 +0,0 @@
-.App {
```
Comment (Alfredbis29) [inline src/App.css:1]: - [ ] While I appreciate the effort and time you have put into this but there is something you need to work on for your project to be completed,  the app is running well but I noticed that it is throwing an error, below :point_down: 
Screenshot from 2023-04-08 22-02-25

_note: In order to ensure that the app is functioning optimally, I recommend fixing this error as soon as possible. The error message indicates that the file `./App.css` could not be found in the specified directory, which is causing the application to fail to compile. To resolve this issue, please make sure that the file exists in the correct location and that the import statement in your `./App.jsx` file is correctly referencing the file._

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @lorenzoworx 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
 :heavy_check_mark: Linters are passing.
:heavy_check_mark: You have a good code structure.
:+1:  Good Job so far
 ## Required Changes 

_Check Under the Review

If you need any suggestion from me I suggest you to use tabindex in your headings paragraph tags to make text . :upside_down_face:
 
 ## Optional suggestions 

- [OPTIONAL ] I noticed that you used the ".jsx" extension in your React Bookstore project. For compatibility reasons, I recommend you to use  ".js" extension instead. Please update your project accordingly. If you need help, let me know. :wink: 

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (BenMukebo) [reply src/App.css:1]: Well said @Alfredbis29 👍🏻

**Review by BenMukebo — APPROVED** [review]
### Hi @lorenzoworx, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- React Router has been correctly set up in <App> compoent. ✔️  
- Descriptive README file. ✔️  


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly it will be better to consider the OPTIONAL request made the previous revioiwer 👇🏻 this is the react naming convention to use .js instad of .jsx

> I noticed that you used the ".jsx" extension in your React Bookstore project. For compatibility reasons, I recommend you to use ".js" extension instead. Please update your project accordingly. If you need help, let me know.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**